python/ray/data/preprocessors/normalizer.py

```diff
@@ -7,16 +7,69 @@
 
 
 class Normalizer(Preprocessor):
-    """Normalize each record to have unit norm.
+    r"""Scales each sample to have unit norm.
 
-    Supports the following normalization types:
-        * l1: Sum of the absolute values.
-        * l2: Square root of the sum of the squared values.
-        * max: Maximum value.
+    This preprocessor works by dividing each sample (i.e., row) by the sample's norm.
+    The general formula is given by
+
+    .. math::
+
+        s' = \frac{s}{\lVert s \rVert_p}
+
+    where :math:`s` is the sample, :math:`s'` is the transformed sample, and :math:`p`
+    is the norm type.
+
+    The following norms are supporting:
+
+        * :math:`L^1`: Sum of the absolute values.
+        * :math:`L^2`: Square root of the sum of the squared values.
+        * :math:`L^\infty`: Maximum value.
 
     Args:
-        columns: The columns that in combination define the record to normalize.
-        norm: "l1", "l2", or "max". Defaults to "l2"
+        columns: The columns to include in samples.
```

Comment:
Wasn't really sure how to explain this clearly. It's confusing how the operation is applied to rows but we specify columns

Reply:
Yeah.... maybe something like "For each row, these the values of these columns will together constitute the sample to scale".

Reply:
Fixed